src/featureMatching/utils/getGaussianPoints.ts

```diff
@@ -84,7 +84,7 @@ export function getGaussianValues(
 
   const result = array.map((value) => {
     const rounded = Math.round(value);
-    return clamp(-width, width)(rounded);
+    return getClampFromTo(-width, width)(rounded);
```

Comment:
You can avoid recreating the function for every value if you put it in a variable before calling array.map.